Makefile

```diff
@@ -35,6 +35,9 @@ test: build
 	go test -coverprofile=coverage.out ./...
 	# go tool cover -html=coverage.out
 
+lint:
+	if [ "`go fmt ./... | wc -l`" -gt 0 ]; then exit 1; fi
```

Comment:
On second thought, this should be `gofmt -l ./...`; will update.

Reply:
Maybe I should just remove `lint` from the default target and just have CI run this?

Reply:
I've opted to drop the Makefile changes and only run the check in CI.